fixtures/artDemo.storage.json

```diff
@@ -0,0 +1,28 @@
+{
+  "artDemo.account": {
+    "transaction": false,
+    "partition-key": [
+      "account_id"
+    ],
+    "clustering-key": [
```

Comment:
You haven't used the clustering key in this PR, but please get familiar with it since it is one of the tricky parts of Scalar DB data model.

Reply:
Thank you. I will do some more reading on this and add a quick sample for this as well. I did already read on the difference between all the keys in Cassandra though.  But yes, it seemed a bit confusing and tricky at first.

Reply:
I updated the demo application and added the usage of a clustering key to be able to `list art for a specific account_id` via scan